go.mod

```diff
@@ -95,4 +95,7 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
+// TODO: Remove this replacement before merging to dev
+replace github.com/linode/linodego => github.com/linode/linodego v1.21.1-0.20230905223015-1df9dbe40b16
+
```

Comment:
So this PR should target the feature branch?

Reply:
So unfortunately `replace` don't support branch names with `/` in them so I had to pin to commit hash. Fortunately this feature has been merged and released in linodego, so I can drop this replace altogether.

Reply:
Oh, sorry about the confusion. I meant it said remove this before merging to dev but this PR is targeting dev, lol

Reply:
Oh oops, good catch! I've re-targeted it to the feature branch.